app/models/contact.rb

```diff
@@ -62,13 +60,4 @@ def republish
   def set_content_id
     self.content_id ||= SecureRandom.uuid
   end
-
-  def register_contact
-    rummager_id = link.gsub(%r{^/}, '')
-    rummager_presenter = ContactRummagerPresenter.new(self)
-    Services.rummager_client.add_document("contact", rummager_id, rummager_presenter.present)
-
-    presenter = ContactPresenter.new(self)
-    Publisher.publish(presenter)
-  end
```

Comment:
The controller create/update actions are not the only places that save contacts and rely on this behaviour.  Off the top of my head there's `Contact#republish` which calls this method (and is in turn called from some rake tasks) and the `CloneContact` interactor that relies on the `save` on contact to perform these actions.  There may be others.  We'll need to take care to provide alternatives for all of them.